src/main/java/uk/co/caprica/picam/PicamNativeLibrary.java

```diff
@@ -165,7 +166,7 @@ public static Path installLibrary(Path installDirectory, boolean overwrite) thro
                 if ("file".equalsIgnoreCase(protocol)) {
                     return installLibraryFromPath(Paths.get(containerUrl.toURI()), installDirectory, overwrite);
                 } else if ("jar".equalsIgnoreCase(protocol)) {
-                    try (FileSystem fs = FileSystems.newFileSystem(containerUrl.toURI(), Collections.<String, Object>emptyMap())) {
+                    try (FileSystem fs = FileSystems.newFileSystem(containerUrl.toURI(), Collections.emptyMap())) {
```

Comment:
Probably OK but depends on JDK/language level whether that inference can be made doesn't it?